frontend/src/modules/entitydetails/components/Metadata.css

```diff
@@ -49,3 +49,13 @@
     margin: 0;
     padding: 0;
 }
+
+.metadata .resource a:nth-child(2)::before {
```

Comment:
Let's add a `className` and use that instead of `nth-child`. It's easier to maintain.